hw03_Vazquezsotoj.py

```diff
@@ -0,0 +1,155 @@
+#################################################################################
+# Author: Javier Vazquez Soto
+# Username:Vazquezsotoj
+#
+# Assignment:hw03
+# Purpose: To create something complex like a house,animal, or person
+# Google Doc Link: https://docs.google.com/document/d/1f5hiQ8grzoAFKAAftUvUoYdcgWC0BoQAqeDspi7zWOU/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#T03_Function_house.py
+#Hw02-loopy-turtles-loopy-languages.py
+#################################################################################
+
+import turtle
+
```

Comment:
Pay attention to the gray squigglies, they indicate style guidelines. This one says you need two blank lines before all function definitions. Fix this everywhere.